src/pages/workspaces/workspace/Data.js

```diff
@@ -1076,7 +1054,7 @@ const WorkspaceData = _.flow(
               setSelectedData({ type: workspaceDataTypes.entities, entityType: tableName })
             })
           })],
-          [workspaceDataTypes.wds, () => wdsDataTableProvider && wdsProxyUrl && wdsSchema && h(WDSContent, {
+          [workspaceDataTypes.wds, () => wdsDataTableProvider && proxyUrlLoaded && wdsSchema && h(WDSContent, {
```

Comment:
Is the `wdsSchema` condition here behaving as desired? While loading the schema, the `wdsSchema` state variable is set to `[]`, which is truthy.

Reply:
I don't think this is desired, checking on this now-

Reply:
It should be ok either way, since the list is empty when it renders the table, and the case shouldn't occur (we should only end up with an empty list schema in an error case). Just to be explicit, I've change the check to !_.isEmpty()